packages/icons/usage.md

```diff
@@ -34,7 +34,7 @@ The standard icon height in ems matches the 19px rendered canvas of text
 set in Avenir Next sized at 16px (19/16 = 1.1875).
 
 ```
-@cf-icon-height: 1.1875em;
+@@cfpb/icon-height: 1.1875em;
```

Comment:
Oops, we don't want to be remaining variables with the scope, and the double at symbol here is definitely wrong.

Reply:
Bloops.